modules/seqexec/server/src/main/scala/seqexec/server/SeqTranslate.scala

```diff
@@ -423,26 +326,32 @@ class SeqTranslate(site: Site, systems: Systems[IO], settings: TranslateSettings
     stepType: StepType,
     sys: InstrumentSystem[IO]
   ): TrySeq[List[System[IO]]] = {
+    def celestialObjectSys(inst: Instrument) =
+      List(
+        Tcs.fromConfig(systems.tcs, inst.hasOI.fold(allButGaos, allButGaosNorOi), None, sys, systems.guideDb)(
+          config,
+          LightPath(TcsController.LightSource.Sky, sys.sfName(config)),
+          extractWavelength(config)),
+        Gcal(systems.gcal, site == Site.GS))
+
     stepType match {
-      case CelestialObject(inst) =>  (sys :: List(
-          Tcs.fromConfig(systems.tcs, inst.hasOI.fold(allButGaos, allButGaosNorOi), None, sys, systems.guideDb)(
-            config,
-            LightPath(TcsController.LightSource.Sky, sys.sfName(config)),
-            extractWavelength(config)),
-          Gcal(systems.gcal, site == Site.GS)
-      )).asRight
+      case StepType.CelestialObject(inst) =>
+        (sys :: celestialObjectSys(inst)).asRight
+
+      case StepType.NodAndShuffle(inst) =>
```

Comment:
This is the only semantically relevant change. It allows us to execute N&S steps though they will behave as regular steps. Previously an attempt to run would cause an error